Workbooks/AKS/Subnet IP Usage/Subnet IP Usage.workbook

```diff
@@ -170,21 +170,25 @@
             "type": 3,
             "content": {
               "version": "KqlItem/1.0",
-              "query": "InsightsMetrics\n| where TimeGenerated < {timeRange:end} and TimeGenerated >= {timeRange:start}\n| where Name == \"cx_ipam_total_ips\"\n| where Namespace == \"prometheus\"\n| extend jsonprop = parse_json(Tags)\n| extend HostName = jsonprop.hostName\n{hostwhereclause}\n| extend podnet_arm_id = jsonprop.podnet_arm_id\n| project TimeGenerated, Val, podnet_arm_id, HostName\n| summarize count = count(Val) by bin(TimeGenerated, {timeRange:grain}), tostring(HostName)\n\n",
+              "query": "InsightsMetrics\n| where TimeGenerated < {timeRange:end} and TimeGenerated >= {timeRange:start}\n| where Name == \"cx_ipam_total_ips\"\n| where Namespace == \"prometheus\"\n| extend jsonprop = parse_json(Tags)\n| extend HostName = jsonprop.hostName\n{hostwhereclause}\n| project TimeGenerated, Val, HostName\n| summarize count = count(Val) by bin(TimeGenerated, {timeRange:grain}), tostring(HostName), Val\n\n",
               "size": 0,
               "timeContext": {
-                "durationMs": 86400000
+                "durationMs": 2592000000
               },
               "queryType": 0,
               "resourceType": "microsoft.containerservice/managedclusters",
               "visualization": "categoricalbar",
               "chartSettings": {
                 "xAxis": "TimeGenerated",
+                "yAxis": [
+                  "Val"
+                ],
                 "group": "HostName",
-                "createOtherGroup": null
+                "createOtherGroup": null,
+                "showLegend": true
               }
             },
-            "name": "query - 1"
+            "name": "IPs allocated from Subnet"
```

Comment:
can you also please share screen shot for query changes ? (before & after) ?

Reply:
Included in a OneNote. Shared it in the main commit comment.

Reply:
Thanks. i get access denied for that one note.